web/docs/use-vast/ingest/README.md

```diff
@@ -1,31 +1,26 @@
 # Ingest
 
-*Ingesting* data refers to the import pipeline that covers the following steps:
+Sending data to VAST (aka *ingesting*) involves spinning up a VAST client node
+that parses and ships the data to a running VAST server node:
 
-```mermaid
-flowchart LR
-  classDef stage fill:#00a4f1,stroke:none,color:#eee
+![Ingest process](/img/ingest.light.png#gh-light-mode-only)
+![Ingest process](/img/ingest.dark.png#gh-dark-mode-only)
```

Comment:
The partition builder architecture is still quite a ways out. Not sure if this makes sense to put into the docs yet.

Reply:
Conceptually it's there, just happens at a different location. I think it's okay to instil this mental model at the user already now.